### ISSUE TITLE
Pause button does nothing

### ISSUE THREAD

**Issue by omicronns** [WebFreak001/code-debug#82]
I tried it on different configurations [windows + arm-none-eabi-gdb (7.10.1)], [virtualbox hosted linux + gdb (7.11.1)]. Debugging starts, but pause button does nothing.

launch.json for arm-none-eabi-gcc:

    {
        "version": "0.2.0",
        "configurations": [
            {
                "name": "Debug",
                "type": "gdb",
                "request": "attach",
                "gdbpath": "arm-none-eabi-gdb-py.exe",
                "remote": true,
                "executable": "./test-elf.elf",
                "target": ":3333",
                "cwd": "${workspaceRoot}",
                "autorun": [
                    "interrupt",
                    "monitor arm semihosting enable",
                    "monitor reset halt",
                    "load",
                    "monitor reset halt",
                    "tb main"
                ]
            }
        ]
    }

**Comment by WebFreak001** [comment]
Yeah I noticed that that happens to some of my projects too. However sometimes it works and sometimes it doesn't. Not really sure how to fix this
